build.py

```diff
@@ -1037,7 +1037,11 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu,
 ENV LD_LIBRARY_PATH /opt/tritonserver/backends/onnxruntime:${LD_LIBRARY_PATH}
 '''
 
-    ort_dependencies = "libgomp1" if 'onnxruntime' in backends else ""
+    # libgomp is needed by both onnxruntime and pytorch backends
+    openmp_dependencies = ""
```

Comment:
just declare a variable "backend_dependencies". Then keep adding the dependencies in this conditioned on whether onnxruntime, or pytorch is a part of backends. It will be more scalable this way.

Reply:
Done.